### ISSUE TITLE
streaming encrypt/decrypt

### ISSUE THREAD

**Issue by boby1975** [FlowCrypt/flowcrypt-ios#230]
Hi
Does anyone know if is it possible in this project do "streaming encrypt and decrypt" of large files according openPGP?
Because I still can’t find a proper working solution for this task for my iOS app (swift).

Thanks in advance.

**Comment by tomholub** [comment]
It's technically possible. As for our app, we have not implemented streaming yet, but eventually we will.

**Comment by boby1975** [comment]
Thanks for the info. As far as I understand you are using OpenPGP.js v4.6.0. It seems that there has already appeared streaming encryption and compression.

**Comment by tomholub** [comment]
Yes, but we're not taking advantage of these things yet. I'll mark this for later, there's quite a few things we have to do before we can implement streaming.

**Comment by tomholub** [comment]
part of #275
